cartography/intel/aws/waf.py

```diff
@@ -37,14 +37,32 @@ def get_waf_classic_web_acls(boto3_session: boto3.session.Session) -> List[Dict]
         return web_acls
 
 
+@timeit
+def get_waf_classic_details(boto3_session: boto3.session.Session, web_acl_id: str) -> Dict:
+
+    response = {}
+    try:
+        client = boto3_session.client('waf')
+        response = client.get_web_acl(WebACLId=web_acl_id)
+    except ClientError as e:
+        logger.error(f"Error retrieving Web ACL {web_acl_id}: {e}")
+
+    return response.get("WebACL", {})
+
+
 @timeit
 def transform_waf_classic_web_acls(web_acls: List[Dict]) -> List[Dict]:
+    boto3_session = boto3.Session()
```

Comment:
this is not correct. we already have a boto3_session passed to the sync method. we should use that.